packages/react-instantsearch-hooks-dom/src/widgets/CurrentRefinements.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useCurrentRefinements } from 'react-instantsearch-hooks';
+
+import { CurrentRefinements as CurrentRefinementsUiComponent } from '../ui/CurrentRefinements';
+
+import type { CurrentRefinementsProps as CurrentRefinementsUiComponentProps } from '../ui/CurrentRefinements';
+import type { UseCurrentRefinementsProps } from 'react-instantsearch-hooks';
+
+export type CurrentRefinementsProps = Omit<
+  CurrentRefinementsUiComponentProps,
+  'disabled' | 'onClick' | 'items' | 'hasRefinements'
+> &
+  UseCurrentRefinementsProps;
+
+export function CurrentRefinements(props: CurrentRefinementsProps) {
+  const {
+    includedAttributes,
+    excludedAttributes,
+    transformItems,
+    ...otherProps
+  } = props;
```

Comment:
```suggestion
export function CurrentRefinements({
    includedAttributes,
    excludedAttributes,
    transformItems,
    ...props
  }: CurrentRefinementsProps) {
    // and using props instead of otherProps lower
```

what do you think about avoiding naming props, as it doesn't get used?

Reply:
Yup! I moved the destructuring in the signature in 052ec448.